extensions/amp-ad/0.1/test/test-amp-ad.js

```diff
@@ -136,9 +136,7 @@ describes.realWin('Ad loader', {amp: true}, env => {
         meta.setAttribute('name', 'amp-3p-iframe-src');
         meta.setAttribute('content', 'https://example.com/remote.html');
         doc.head.appendChild(meta);
-        a4aRegistry['zort'] = () => {
-          throw new Error('predicate should not execute if remote.html!');
-        };
+        a4aRegistry['zort'] = (win, element, useRemoteHtml) => !useRemoteHtml;
```

Comment:
Could we add test coverage that ensures useRemoteHtml value is correct given various conditions of the page (e.g. remote HTML present, network does not allow remote HTML, etc).